boss-ws/src/main/java/org/genomebridge/boss/http/resources/FCSResource.java

```diff
@@ -2,25 +2,33 @@
 
 import java.util.concurrent.ConcurrentHashMap;
 
-import javax.ws.rs.DELETE;
-import javax.ws.rs.GET;
-import javax.ws.rs.HEAD;
-import javax.ws.rs.HeaderParam;
-import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
+import javax.ws.rs.*;
```

Comment:
Agree, I updated only on FCSResource, but I'll take in consideration. Thank you!